src/symbolize/gimli/parse_running_mmaps_unix.rs

```diff
@@ -78,6 +93,12 @@ impl MapsEntry {
     }
 }
 
+#[cfg(not(target_os = "nto"))]
+pub fn config() -> (&'static str, usize) {
+    ("/proc/self/maps", 0)
+}
```

Comment:
This moves to runtime what can be known at compile-time. Please describe a `static` and a `const` instead, with appropriate names. If you wish, you can put them in a `mod config {}`.